src/utils/daos_hdlr.h

```diff
@@ -62,6 +66,25 @@ enum obj_op {
 	OBJ_DUMP
 };
 
+#define NUM_DIRENTS 24
+#define MAX_FILENAME 256
+
+struct dfs_daos_t {
+	dfs_obj_t* dir;
+	struct dirent ents[NUM_DIRENTS];
+	daos_anchor_t anchor;
+	uint32_t num_ents;
+};
+
+typedef struct file_dfs_t {
+	enum {POSIX, DAOS} type;
+	int                  fd;
+	/* DAOS specific variables for I/O */
+	daos_off_t offset;
+	dfs_obj_t* obj;
+	dfs_t* dfs;
+} file_dfs_t;
```

Comment:
do we need the above 2 structs in this header? can we just have them local to the file you are using them?

Reply:
Agreed. This is also causing a compile issue (noted previously)

Reply:
@mchaarawi yeah I will move that.